production/helm/loki/values.yaml

```diff
@@ -201,6 +209,12 @@ loki:
 
   # -- Check https://grafana.com/docs/loki/latest/configuration/#schema_config for more info on how to configure schemas
   schemaConfig: {}
+  
+  # -- Check https://grafana.com/docs/loki/latest/configuration/#limits_config for more info on how to configure limits
+  limitsConfig: {}
+  
+  # -- Check https://grafana.com/docs/loki/latest/configuration/#compactor for more info on how to configure compactor
+  compactorConfig: {}
```

Comment:
@trevorwhitney do we want to move all configuration into values or rather move all but the most important into `config`?